flexmeasures/ui/crud/assets/views.py

```diff
@@ -362,9 +362,24 @@ def properties(self, id: str):
 
         asset_form.process(data=process_internal_api_response(asset_dict))
 
+        asset_summary = {
+            "Name": asset.name,
+            "Latitude": asset.latitude,
+            "Longitude": asset.longitude,
+            "Parent Asset": (
```

Comment:
I would like to summarize parent asset info in one row, maybe even clickable, like  `<a href="/assets/ParentID">Asset name (type)</a>`